plugins/modules/cloud/misc/terraform.py

```diff
@@ -360,7 +385,13 @@ def build_plan(command, project_path, variables_args, state_file, targets, state
         return plan_path, False, out, err, plan_command if state == 'planned' else command
     elif rc == 1:
         # failure to plan
-        module.fail_json(msg='Terraform plan could not be created\r\nSTDOUT: {0}\r\n\r\nSTDERR: {1}'.format(out, err))
+        module.fail_json(
+            msg='Terraform plan could not be created\r\nSTDOUT: {0}\r\n\r\nSTDERR: {1}\r\nCOMMAND: {2}'.format(
+                out,
+                err,
+                ' '.join([shlex_quote(arg) for arg in variables_args])
```

Comment:
Why change only this `fail_json` call, and why change it only partially (the main part of the command, namely `plan_command`, isn't there)?

Reply:
That me being selfish and not thinking of the wider user base :), trying to take MVP to a new level :). I'm fixing both the options... and any other where it makes sense.

Reply:
Done.